Documentation/operations/upgrade.rst

```diff
@@ -403,6 +403,8 @@ Helm Options
   values are kept, ``ipam.operator.clusterPoolIPv4PodCIDRList`` now defaults to
   a singleton containing the default CIDR value for the removed value
   ``ipam.operator.clusterPoolIPv4PodCIDR`` (and similarly for IPv6).
+* Values ``proxy.prometheus.enabled`` & ``proxy.prometheus.port`` are deprecated in favor of
```

Comment:
nit
```suggestion
* Values ``proxy.prometheus.enabled`` and ``proxy.prometheus.port`` are deprecated in favor of
```

Reply:
@zacharysarah thanks - fixed in the latest update